frameworks/helloworld/src/main/dist/examples/uri.yml

```diff
@@ -6,8 +6,9 @@ pods:
   hello:
     count: {{HELLO_COUNT}}
     uris:
-      - "https://github.com/mesosphere/dcos-commons/blob/master/README.md"
-      - "https://github.com/mesosphere/dcos-commons/blob/master/docs/generate.sh"
+      - https://github.com/mesosphere/dcos-commons/blob/master/README.md
```

Comment:
Do we exercise these changes in a test of some sort?

Reply:
Yes. See ServiceSpecTest in hello-world 's unit tests.

Reply:
I ask because they have changed, and didn't see a corresponding change here (but I may have just missed it). OK. Found id. We basically just check that we're able to build the `DefaultScheduler` with the specification and don't check any specific values, right?